resources/views/frontend/articles/show.blade.php

```diff
@@ -108,7 +108,7 @@
             </div>
             <div class="row">
                 <div class="col-md-8 col-md-push-2 long-text">
-                    {!! $article->content !!}
+                    {!! $article->getParsedContent() !!}
                 </div>
```

Comment:
In case you renamed the method to `getParsedContentAttribute()` 
you can use:

```suggestion
                    {!! $article->parsed_content !!}
                </div>
```